### DISCUSSION TITLE
rename editor.file-picker.hidden to ignore-hidden

### DISCUSSION THREAD

**Discussion by tshepang** [helix-editor/helix#7280] [General]
That will make it more clear what it actually does

**Comment by pascalkuthe** [comment]
We dont do breaking config changes (or any renames in general) when it's not absolutely necessary

**Reply by the-mikedavis** [reply]
Also see https://github.com/helix-editor/helix/issues/3086

**Reply by tshepang** [reply]
it could be an alias, and the old name deprecated
